en-US/Docs/WhatsNew.md

```diff
@@ -27,33 +27,33 @@ ___
 
 The new Microsoft IoT Grove Kit is an easy to use solution for Makers, as well as a powerful prototyping platform for IoT Startups and Enterprise Innovators. You can pre-order today at [Seeed Studio Bazaar](https://www.seeedstudio.com/Microsoft-IoT-Grove-Kit-p-2694.html) and [Digi-Key](http://www.digikey.com/en/product-highlight/s/seeed/microsoft-iot-grove-kit?WT.z_cid=sp_1597_MIot).
 
-### Refreshed Adafruit IoT Starter Pack for Raspberry Pi 3
+### [Refreshed Adafruit IoT Starter Pack for Raspberry Pi 3]({{site.baseurl}}/{{page.lang}}/Docs/AdafruitWelcome.htm)
 ![Adafruit starter pack]({{site.baseurl}}/Resources/images/AdafruitStarterPack/AdafruitMakerKitClosedSmall.jpg) 
 
 Adafruit has [updated their starter pack](https://www.adafruit.com/product/2733) to now include Raspberry Pi 3 and Windows 10 IoT Core.
 
 
 ## New major features
 ___ 
-* Windows IoT Remote Client - remote into your IoT device to control and view what is displayed on your IoT device from your desktop or phone
-* Store integration - connect Windows 10 IoT Core to the store to service applications 
-* NOOBS - 8GB cards now supported 
-* Better Azure IoT Hub connectivity - provision your device with a device identity in the cloud 
+* [Windows IoT Remote Client]({{site.baseurl}}/{{page.lang}}/Docs/RemoteDisplay.htm) - remote into your IoT device to control and view what is displayed on your IoT device from your desktop or phone
+* [Store integration]({{site.baseurl}}/{{page.lang}}/Docs/Store.htm) - connect Windows 10 IoT Core to the store to service applications 
+* [NOOBS]({{site.baseurl}}/{{page.lang}}/Docs/GetStarted/noobs/getstartedstep2.htm) - 8GB cards now supported 
+* Better [Azure IoT Hub]({{site.baseurl}}/{{page.lang}}/Docs/cloudintro.htm) connectivity - provision your device with a device identity in the cloud 
 * [Shell capabilities](https://github.com/ms-iot/samples/tree/develop/IoTHomeAppSample) - multiple foreground app support
 * [IoTivity AllJoyn Device System Bridge](https://github.com/ms-iot/samples/tree/develop/AllJoyn/Samples/OICAdapter)
-* Arduino INO integration - work better with Arduino hardware with this command line toolkit
+* [Arduino INO integration](http://inotool.org/) - work better with Arduino hardware with this command line toolkit
 
-### Improved documentation
-* Updated samples page - new tagging system to sort through different samples 
-* Updated documentation page - added a table of contents (finally) and new documentation
+### [Improved documentation]({{site.baseurl}}/{{page.lang}}/Docs.htm)
```

Comment:
If we link to the docs page below, do we need to link the heading?  I usually prefer to leave headings unlinked unless they're necessary